SalesforceCommerceCloud/pwa-kit
# W-12627197 -v3/template-retail-app Indicate live regions for dynamically changing content

# Description
This PR updates the code to make sure dynamic content is visible to screen reader in cart product-item and order summary for v3 retail app template
**NOTE:** When you test this PR, there is a current bug in Recommendation Component that caused it to re-render every time shopper changes basket item quantity in cart page. As a result, the screen reader will read it out loud following a quantity changes which can be a lot of noise. 
A PR will be created to address that issue separately from this PR

https://github.com/SalesforceCommerceCloud/pwa-kit/assets/52219283/eaf5b1a1-57e7-4120-a1d2-0f2dbe799dca




# Types of Changes




> Breaking changes include:
>
> - Removing a public function or component or prop
> - Adding a required argument to a function
> - Changing the data type of a function parameter or return value
> - Adding a new peer dependency to `package.json`

# Changes

- (change1)

# How to Test-Drive This PR

- Checkout code
- Run Retail App template
- Add a few items in cart/add item to wishlist
- Go to Cart Page.
- Open VoiceOver (Cmd + f5)
- Change the quantity
- Observe the screen reader is now reading out loud the changes in quantity, prices and order summary section

# Checklists




## General


## Accessibility Compliance

You must check off all items in **one** of the follow two lists:


_or..._


## Localization
